index.js

```diff
@@ -109,6 +122,46 @@ class RenderPDF {
         renderer.killChrome();
     }
 
+    async renderPdfFromHtml(html, options) {
```

Comment:
This is almost the same function as `renderPdf` common functionality should be extracted to common place,
maybe something like `renderPdfCb((page) => {...}, options)`